### DISCUSSION TITLE
Welcome to trouble.nvim Discussions!

### DISCUSSION THREAD

**Discussion by folke** [folke/trouble.nvim#288] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by donnySimba** [comment]
How to operate it

**Reply by TimothyElems** [reply]
After you install it using your nvim package manager, just type `:Trouble`

**Comment by TimothyElems** [comment]
@folke Thanks man! I've tried using nvim for a few years, and it's only cause of this plugin that I can finally switch from vsc*de 😂
Huge Thanks!

**Comment by omcdrucifer** [comment]
I can't add configurations to trouble. For context I am new to lua and lazy, and a lot of things. 

I am using the method of /config and /plugins. I created a .lua and pasted the install code into it and wrapped it in a return. That at least gets trouble into lazy. 

The problem is I cannot add any of the default settings code. I tried copying to clipboard with the copy button in git. I then went into the code for trouble and pasted everything in the opts = {} field at the top. But whenever I do that, lazy asks me to purge/clean trouble. It only works if I leave the install code alone and change nothing. 

I need help understanding how configuring this works. I again know nothing about lua and have basically been riding on lazy's copy/paste so if something isn't perfectly written for copy/paste, it won't work. It took me a lot of time and frustration just to figure out that everything gets wrapped in a return because I was going into this knowing 0% about lua.
